components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/data/includes/center_plugin.thumbs.js.html

```diff
@@ -810,6 +811,7 @@
         $("div#page").removeData('page');
         $contentDetails.removeData('field');
     };
+    OME.clearThumbnailsPanel = clearThumbnailsPanel;
```

Comment:
A comment in the code here would help remind us why this is made globally available
// Allows other web apps to access this method